packages/server/src/automations/tests/n8n.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { getConfig, afterAll, runStep, actions } from "./utilities"
+
+describe("test the outgoing webhook action", () => {
+  let config = getConfig()
+
+  beforeAll(async () => {
+    await config.init()
+  })
+
+  afterAll()
+
+  it("should be able to run the action and default to 'get'", async () => {
+    const res = await runStep(actions.n8n.stepId, {
+      url: "http://www.test.com",
```

Comment:
test.com is a privately owned domain, I tend to avoid it in tests just in case. It could theoretically point anywhere and if we do accidentally hit it there's no telling what it'll do.

I prefer to use example.com, which is owned by IANA and is a specifically reserved domain: https://www.rfc-editor.org/rfc/rfc2606.